lib/models/tic_tac_toe_game.rb

```diff
@@ -95,13 +106,18 @@ def self.name
       end
 
       def pretty_print
-        x = @board.collect {|r| r.collect { |x| PRETTY_MAP[x] } }
+        x=@board.collect {|r| r.collect { |x| PRETTY_MAP[x] } }
         """
-        _______
-        |#{x.map{|y| y.join('|')}.join('|\n')}|
+        #{@state}
+        _____
+        |#{x[0][0]}|#{x[1][0]}|#{x[2][0]}|
```

Comment:
so my plan didn't work, huh

Reply:
not on the first try but we could swap it back in and see.